### PR TITLE
WEBEU2 - UI III Flexbox - James Eneh 🇳🇬

### PR DESCRIPTION
**Pull request by erozonachi**

### CODE REVIEW

**Comment by erozonachi** [comment]
Thanks, @maximesalomon for the PR Title correction 👍🏽

File: great-idea/index.html [hunk 1]
```diff
@@ -1,23 +1,102 @@
...
+        </div>
+        <div class="header-img item"></div>
+      </section>
+    </header>
+    <hr>
+
+    <main class="container stack">
+      <section class="container space-btw">
+        <div class="wide-6">
+          <h2>Features</h2>
```
Comment (maximesalomon) [inline great-idea/index.html:50]: I would put the `<h2>` into a `<h3>` here and on the others below

File: great-idea/about.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!doctype html>
```
Comment (maximesalomon) [inline great-idea/about.html:1]: Good job working on some stretch :)

File: great-idea/services.html [hunk 3]
```diff
@@ -3,8 +3,9 @@
 <html lang="en">
```
Comment (maximesalomon) [inline great-idea/services.html:3]: Good job on the services page :)

**Review by maximesalomon — COMMENTED** [review]
Good job on this project James! HTML looks good, I would advise that you try to be as close as possible to the design for CSS.

But you got this 👍

**Pull request merged**